### ISSUE TITLE
(Interactive Description) Black reading block highlights appear in Preferences menu

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#518]
While reviewing #490 and #491, I noticed that the black focus highlights for reading blocks were showing up in the Preferences Menu when using keyboard navigation with or without Interactive Highlights turned on. This is only with the keyboard. They black highlights do not appear at all with mouse with or without Interactive Highlights, which seems correct.

**Comment by jessegreenberg** [comment]
Thanks @terracoda I will check it out.

**Comment by jessegreenberg** [comment]
This is not an issue in the JT RC, so something must have changed recently to cause this.

**Comment by jessegreenberg** [comment]
It isn't just the JT RC SHAs, this isn't happening at all in JT. So some configuration there must be preventing it from happening.

**Comment by jessegreenberg** [comment]
Caused by scenery commit a6761061545cc92d27da8c36af7c76615c234523, initial value for speechAllowedAndFullyEnabledProperty was incorrect. @terracoda this has been fixed.
